dhcpv6/ztpv6/parse_vendor_options_test.go

```diff
@@ -99,3 +100,50 @@ func TestParseVendorDataWithVendorClass(t *testing.T) {
 		})
 	}
 }
+
+func TestParseVendorDataWithClientId(t *testing.T) {
```

Comment:
Can this test be unified with the previous one? It is also parsing the vendor class after all.

Reply:
Will try to merge them :-)